### ISSUE TITLE
[core] Remove warning from RouterContext

### ISSUE THREAD

**Issue by palamago** [Datawheel/canon#1229]
When `npm run dev` there is a warning part of  `RouterContext` 

``` 
Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

* Move code from componentWillMount to componentDidMount (preferred in most cases) or the constructor.

Please update the following components: RouterContext
    in RouterContext
    in div
    in div
    in CanonProvider
    in Connect(CanonProvider)
    in Provider
    in Context.Provider
    in I18nextProvider
    in Context.Provider
    in HelmetProvider
```

**Comment by davelandry** [comment]
Might be dependent/solved by #672 😬

**Comment by jhmullen** [comment]
Closing as duplicate
